### PR TITLE
Address a number of Gogland code inspection issues so I don't need to look  at so many squiggles.

### PR DESCRIPTION
**Pull request by geeknoid**
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/384)

### CODE REVIEW

File: cmd/server/inventory.go [hunk 1]
```diff
@@ -96,7 +96,7 @@ func listBuilders() error {
 
 func printConfigValidator(v pkgadapter.ConfigValidator) {
 
-	fmt.Printf("Params: \n")
+	fmt.Print("Params: \n")
```
Comment (douglas-reid) [inline cmd/server/inventory.go]: should we be using fmt.Print at all here? aren't we supposed to be on glog?

File: pkg/aspect/quotasManager_test.go [hunk 2]
```diff
@@ -193,7 +194,8 @@ func TestQuotaWrapper_Execute(t *testing.T) {
 		{goodMd, 1, nil, false, errEval, make(map[string]o), "expected"},
 		{goodMd, 1, nil, false, labelErrEval, make(map[string]o), "expected"},
 		{goodMd, 1, nil, false, goodEval, map[string]o{"request_count": {1, []string{"source", "target"}}}, ""},
-		{goodMd, 0, fmt.Errorf("alloc-forced-error"), false, goodEval, map[string]o{"request_count": {1, []string{"source", "target"}}}, "alloc-forced-error"},
+		{goodMd, 0, errors.New("alloc-forced-error"), false, goodEval, map[string]o{"request_count": {1, []string{"source", "target"}}},
```
Comment (douglas-reid) [inline pkg/aspect/quotasManager_test.go]: nit: this looks weird with the new line break.  should we move the `map[string]o{"request_count": {1, []string{"source", "target"}}}` up (as it is used in another test case, too)?

File: cmd/server/inventory.go (hunk 1 above)
Comment (geeknoid) [inline cmd/server/inventory.go]: The two front-ends(mixc and mixs) use normal stdout stuff. The core mixer uses glog.

File: pkg/aspect/quotasManager_test.go (hunk 2 above)
Comment (geeknoid) [inline pkg/aspect/quotasManager_test.go]: Done.

**Comment by geeknoid** [comment]
PTAL.

File: cmd/server/inventory.go (hunk 1 above)
Comment (douglas-reid) [inline cmd/server/inventory.go]: Then should we not use glog in the main.go for server (https://github.com/istio/mixer/blob/master/cmd/server/main.go#L21)?

We probably want consistency there. And, we should some mention of the exception (with reasoning) in our [docs](https://github.com/istio/mixer/blob/master/doc/dev/conventions.md#logging-conventions).

**Comment by geeknoid** [comment]
Consolidated out stdout & stderr are handled in mixc and mixs. It's all consistent now, and if we decide to change the output to something else, it's all centralized. That can be a separate issue if we find it useful.

PTAL.

**Pull request merged**